webpack.config.js

```diff
@@ -24,7 +24,7 @@ const devEntryPoints = isProduction
   ? []
   : [
       "react-hot-loader/patch",
-      "webpack-dev-server/client?http://localhost:8080",
+      "webpack-dev-server/client?http://localhost:9090",
```

Comment:
did you do project wise search for `8080` just to make sure you got all of them? :D